packages/core/integration-tests/test/svg.js

```diff
@@ -0,0 +1,39 @@
+import assert from 'assert';
+import {
+  bundle,
+  removeDistDirectory,
+  distDir,
+  outputFS,
+} from '@parcel/test-utils';
+import defaultConfigContents from '@parcel/config-default';
+import path from 'path';
+
+const config = {
+  ...defaultConfigContents,
+  transforms: {
+    ...defaultConfigContents.transforms,
+    '*.svg': [
+      '@parcel/transformer-svgo',
+      '@parcel/transformer-svg-react',
+      '@parcel/transformer-js',
```

Comment:
replace this with `...` than it will run the Javascript pipeline instead :)

Reply:
I just did it myself, hopefully everything works. So we can ship this 🚀

Reply:
@DeMoorJasper - haha whoops I did it too and didn't realize.

Just curious, what is that magic of `...`?

Reply:
It's based on the spread operator, it tells parcel to look for any additional pipelines if there is any. and place them at the position of the `...` in this case after svgo and svg-react.

I think this is actually no necessary and will automatically run through the js pipeline without the `...`.

See the PR https://github.com/parcel-bundler/parcel/pull/2568 for more info

Reply:
`...` means "continue with the pipeline declared in the `extend`ed config".

In this case, doing `asset.type = "js";` will switch to the `js` pipeline either way, so `...` would be ignored anyway